website/LICENSES

```diff
@@ -0,0 +1,204 @@
+============================== JetBrains/intellij-community ==============================
```

Comment:
Just out of curiosity: What in this folder is apache-licensed by intellij? Maybe we should clarify that?
Because most files just look like config files...

Especially once we produce actual doc that would be under our repo licence (which we don't yet have) and copyright by us.

Reply:
the package svg you found later  
this is the way I saw it in most OSS repos, basically a file with all licenses  
I guess that as long as the files with the copyright mention it, it's fine